core/storage/user/gae_models.py

```diff
@@ -354,6 +354,8 @@ class UserQueryModel(base_models.BaseModel):
     user_ids = ndb.StringProperty(indexed=True, repeated=True)
     # Id of the user who submitted the query.
     submitter_id = ndb.StringProperty(indexed=True, required=True)
+    # Id of the email model having information of sent emails.
```

Comment:
Be specific: give the actual class name of the model. E.g.

Id of the SentEmailModel(?) instance containing information about sent emails.

Also, I suggest Id --> ID (here and above).